test/tests.py

```diff
@@ -1,29 +1,161 @@
 import unittest
 import os
 import sys
+import csv
+import pickle
+import json
 from os import path
-sys.path.insert(0, '../lib')
-sys.path.insert(0, '../in_out')
-sys.path.append(path.dirname(path.dirname(path.abspath(__file__))))
-from lib.analysis import network
-from lib import config, nickTracker
+sys.path.insert(0,'../lib')
+sys.path.insert(0,'../in_out')
+sys.path.append( path.dirname( path.dirname( path.abspath(__file__) ) ) )
+from lib.analysis import network, user, channel
+from lib import config, nickTracker, util
 from in_out import reader
 import networkx as nx
-test_dir = os.path.dirname(__file__)
 
-log_directory = os.path.join(test_dir, 'data/input/')
-channel_name = config.CHANNEL_NAME
-starting_date = config.STARTING_DATE
-ending_date = config.ENDING_DATE
-output_directory = os.path.join(test_dir, 'data/output/')
+dir = os.path.dirname(__file__)
 
+log_directory = os.path.join(dir, 'data/input/')
+output_directory = os.path.join(dir, 'data/output/')
```

Comment:
this is not the output directory ( you aren't putting the outputting anything here). expected_output_directory makes sense